Chapter2/Chapter2-1-1/Song.cs

```diff
@@ -0,0 +1,35 @@
+﻿// 1.2 Songクラスでプロパティとコンストラクタを定義
+namespace Chapter2_1_1 {
+    /// <summary>
+    /// 歌クラス
+    /// </summary>
+    internal class Song {
+
+        /// <summary>
+        /// 曲名
+        /// </summary>
+        public string Title { get; set; }
+
+        /// <summary>
+        /// アーティスト名
+        /// </summary>
+        public string ArtistName { get; set; }
+
+        /// <summary>
+        /// 曲の長さ
+        /// </summary>
+        public int Length { get; set; }
```

Comment:
【FYI: 情報共有。アクション不要です。】
もしこれらのプロパティが後から変更されることがない想定なら、読み取り専用にしてしまったほうが、変更される可能性を考慮せずに済むぶん読みやすくなります。
```suggestion
        /// <summary>
        /// 曲名
        /// </summary>
        public string Title { get; }

        /// <summary>
        /// アーティスト名
        /// </summary>
        public string ArtistName { get; }

        /// <summary>
        /// 曲の長さ
        /// </summary>
        public int Length { get; }
```